### DISCUSSION TITLE
Print elevations of multiple racks

### DISCUSSION THREAD

**Discussion by pbrinkhoff** [netbox-community/netbox#12581] [Q&A]
### NetBox version

v3.5.1

### Feature type

New functionality

### Proposed functionality

I saw #160 but I still can’t find the option to print (or export to pdf) multiple rack views from the elevations tab. Am I right?

### Use case

Easy to export elevation view to pdf

**Comment by freakdetoby** [comment]
This would be very, very helpful! Floorplan is nice, but print out the elevation would be great!
